src/mmw/apps/geoprocessing_api/views.py

```diff
@@ -1537,3 +1591,14 @@ def _parse_aoi(data):
     wkaoi = serializer.validated_data.get('wkaoi')
 
     return area_of_interest, wkaoi
+
+
+def _parse_subbasin_input(request):
+    if 'wkaoi' not in request.data and 'huc' not in request.data:
+        raise ValidationError('Must specify exactly one of: WKAoI ID or HUC')
+
+    serializer = AoiSerializer(data=request.data)
+    serializer.is_valid(raise_exception=True)
+
+    return itemgetter('area_of_interest', 'wkaoi', 'huc')(
+        serializer.validated_data)
```

Comment:
This is Python shorthand for returning multiple values from a dict. See https://stackoverflow.com/a/52083390/6995854